### PR TITLE
An-4539-nfts

### PR DESCRIPTION
**Pull request by eric-laurello**

### CODE REVIEW

File: models/silver/nft/silver__nft_sales_bluemove.sql [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    END AS token_version,
+    event_address AS platform_address,
+    COALESCE(
+        event_data :id.token_data_id.collection,
+        event_data :token_id.token_data_id.collection
+    ) :: STRING AS project_name,
+    COALESCE(
+        event_data :id.token_data_id.name,
+        event_data :token_id.token_data_id.name
+    ) :: STRING AS tokenid,
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_bluemove.sql:250]: nitpicky -- but i think token_id would be consistent with our naming convention

File: models/silver/nft/silver__nft_sales_seashine.sql [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        event_address,
+        event_resource,
+        event_data,
+        event_module,
+        event_type,
+        _inserted_timestamp
+    FROM
+        {{ ref('silver__events') }}
+    WHERE
+        --mercato
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_seashine.sql:29]: does this 'mercato' comment mean the first half of the filter is for sales done through that aggregator?

File: models/silver/nft/silver__nft_sales_topaz.sql [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                'FillCollectionBidEvent',
+                'SellEvent'
+            ) THEN 'bid_won'
+        END AS event_kind,
+        _inserted_timestamp
+    FROM
+        {{ ref('silver__events') }}
+    WHERE
+        --mercato
+        {# (
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_topaz.sql:37]: should this be uncommented?

File: models/silver/nft/silver__nft_sales_seashine.sql [hunk 4]
```diff
@@ -0,0 +1,202 @@
+{{ config(
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_seashine.sql:1]: might have to update the model name to `seashrine`

File: models/silver/nft/silver__nft_sales_combined.sql [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    merge_exclude_columns = ["inserted_timestamp"],
+    tags = ['noncore']
+) }}
+
+WITH all_nft_platform_sales AS (
+
+    SELECT
+        *
+    FROM
+        {{ ref('silver__nft_sales_combined_view') }}
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_combined.sql:15]: Would incremental logic here help optimize the model?

File: models/silver/nft/silver__nft_sales_okx.sql [hunk 6]
```diff
@@ -0,0 +1,252 @@
+{{ config(
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_okx.sql:1]: okx analytics on their site show sales the last few days, but we don't seem to be returning any https://www.okx.com/web3/marketplace/nft/collection/aptos/apts-7

File: models/silver/nft/silver__nft_sales_souffl3.sql [hunk 7]
```diff
@@ -0,0 +1,186 @@
+{{ config(
```
Comment (tarikceric) [inline models/silver/nft/silver__nft_sales_souffl3.sql:1]: This model is returning 0 rows

File: models/silver/nft/silver__nft_sales_bluemove.sql (hunk 1 above)
Comment (eric-laurello) [reply models/silver/nft/silver__nft_sales_bluemove.sql:250]: agreed - was debating changing it, but evm used tokenid

File: models/silver/nft/silver__nft_sales_combined.sql (hunk 5 above)
Comment (eric-laurello) [reply models/silver/nft/silver__nft_sales_combined.sql:15]: got catch

File: models/silver/nft/silver__nft_sales_okx.sql (hunk 6 above)
Comment (eric-laurello) [reply models/silver/nft/silver__nft_sales_okx.sql:1]: I think the recent stuff is all inscriptions

File: models/silver/nft/silver__nft_sales_seashine.sql (hunk 2 above)
Comment (eric-laurello) [reply models/silver/nft/silver__nft_sales_seashine.sql:29]: just need to clean up the comments

File: models/silver/nft/silver__nft_sales_seashine.sql [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+
+{% if is_incremental() %}
+AND _inserted_timestamp >= (
+    SELECT
+        MAX(_inserted_timestamp)
+    FROM
+        {{ this }}
+)
+{% else %}
+    {# AND block_timestamp :: DATE >= '2022-10-19' #}
```
[the remark this answers is not in the document]
Comment (eric-laurello) [reply models/silver/nft/silver__nft_sales_seashine.sql:50]: will delete these

File: models/silver/nft/silver__nft_sales_topaz.sql (hunk 3 above)
Comment (eric-laurello) [reply models/silver/nft/silver__nft_sales_topaz.sql:37]: I will delete the commented code

**Review by tarikceric — APPROVED** [review]

**Pull request merged**